packages/js/product-editor/src/utils/record-product-event.ts

```diff
@@ -1,10 +1,13 @@
 /**
  * External dependencies
  */
-import { Product } from '@woocommerce/data';
+import { Product, ProductVariation } from '@woocommerce/data';
 import { recordEvent } from '@woocommerce/tracks';
 
-export function recordProductEvent( eventName: string, product: Product ) {
+export function recordProductEvent(
+	eventName: string,
+	product: Product | ProductVariation
```

Comment:
It feels really unfortunate that we have to do this sort of thing all over the place.